blocks/lists.ts

```diff
@@ -361,12 +360,10 @@ const LISTS_INDEXOF = {
     if (!operatorsDropdown) throw new Error('field_dropdown not found');
     this.appendValueInput('FIND').appendField(operatorsDropdown, 'END');
     this.setInputsInline(true);
-    // Assign 'this' to a variable for use in the tooltip closure below.
-    const thisBlock = this;
-    this.setTooltip(function () {
+    this.setTooltip(() => {
```

Comment:
Did prettier rewrite this into an arrow function, or did you do that?

Reply:
I did this, to resolve lint errors related to redefining `this`

Reply:
Oh good. I was not stoked about automated rewriting of functions into arrow functions.